### DISCUSSION TITLE
Whats about a API

### DISCUSSION THREAD

**Discussion by ytalexyt033** [ORelio/Minecraft-Console-Client#2251] [Ideas]
I think it would be nice to connect the bot to the discord to receive notifications about the death, the appearance of a nearby player, etc.

**Comment by milutinke** [comment]
There is a discord chat bot which uses webhooks, you can find it in the `config/ChatBots` folder.
You can modify it to have that type of notifications by using corresponding methods from the `ChatBot` class.
